webapp/src/types/index.d.ts

```diff
@@ -0,0 +1,127 @@
+type Label = {
+    id: number;
+    name: string;
+    color: CSS.Properties;
+}
```

Comment:
I would prefer to have these types exported, otherwise there's no way to trace the code within GitHub's interface. Something like `Label` is a vague name, and it's not clear where to find its definition if it's global like this. What do you think?

Also I'm thinking this file could be called `types/github_types.ts`